pkg/login/social/support_bundle.go

```diff
@@ -13,19 +13,19 @@ import (
 )
 
 func (ss *SocialService) registerSupportBundleCollectors(bundleRegistry supportbundles.Service) {
-	for name := range ss.oAuthProvider {
+	for name := range ss.socialMap {
```

Comment:
This change would register all of the providers regardless of enabled/disabled. (we need to register all of the providers for sso setting). 

So the Support bundles page would list all of the providers. To change this behaviour we could add an `EnabledFn`/`ShouldListFn` to the `supportbundles.Collector` struct and call that during the listing, what do you think? @Jguer @IevaVasiljeva

Reply:
We could also return empty content when its not enabled

Reply:
> We could also return empty content when its not enabled

Yes, it's gonna remove it from the bundle, but the main issue is that on the FE all of the providers will be displayed.

Reply:
oh yeah, like what you can select?

Reply:
:+1: @mgyongyosi on adding an EnabledFn, it would probably be good for other services

Reply:
`EnabledFn` has been added to `supportbundles.Collector`, could you please review my changes again? Thanks!